src/style.css

```diff
@@ -0,0 +1,49 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.ListItems {
```

Comment:
[optional] It would be better to use class names in small letters to better use the best HTML best practices.